### DISCUSSION TITLE
first time to create nextjs app with this error ,anyone can help ?

### DISCUSSION THREAD

**Discussion by clarkshu1016** [vercel/next.js#63348] [Help]
### Summary

'Log files:
/Users/xxxx/.npm/_logs/2024-03-16T02_48_56_166Z-debug-0.log

# npm resolution error report

While resolving: nextflix-clone@0.1.0
Found: eslint@undefined
node_modules/eslint
  dev eslint@"^8" from the root project

Could not resolve dependency:
peer eslint@"^7.23.0 || ^8.0.0" from eslint-config-next@14.1.3
node_modules/eslint-config-next
  dev eslint-config-next@"14.1.3" from the root project

Fix the upstream dependency conflict, or retry
this command with --force or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.

**Comment by devjiwonchoi** [comment]
Try `npm up`
